tests/integration/commands/test_init.py

```diff
@@ -64,8 +64,13 @@ def fake_template_dirs(tmp_path):
 )
 @pytest.mark.parametrize("project_dir", [None, "project-dir"])
 @pytest.mark.usefixtures("fake_template_dirs")
-def test_init(app, capsys, monkeypatch, profile, expected_file, project_dir):
+def test_init(app, capsys, monkeypatch, profile, expected_file, project_dir, tmp_path):
```

Comment:
You may just use `empty_working_directory` fixture and you'll not need to create it by yourself